### PR TITLE
Add second blog post for team clojurians

### PR DESCRIPTION
**Pull request by saskali**

### CODE REVIEW

File: blog/_posts/2017-09-04-clojurians-update.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+## Learning and Growing
+
+Through a glorious combination of working on re-frame's tracing tool, reading other peoples' open source projects, hanging out with our wonderful mentors and coaches, and tinkering with our own side projects, we gathered experience in programming with ClojureScript and re-frame and learned about langugage specific idioms. What I find most valuable though is the great advice we got from our coaches and our mentor. They all have given us so much input for how to become good programmers, not only technically, but especially in terms of finding a good and efficient workflow, addressing problems on a conceptual level and the importance of clear and constructive communication.
```
Comment (v-thakkar) [inline blog/_posts/2017-09-04-clojurians-update.md]: Change 'What **I** found' -> 'What **we** found'

File: blog/_posts/2017-09-04-clojurians-update.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+[coaching with Matt!](/img/blog/2016/coaching.jpg)
+*photo taken by Chris Sun*
+
+We are very thankful to have the opportunity to work together so closely with all of you! A big thank you for all your support!
+
+## Building and Collaborating
+
+We worked on many features for the [tracing tool](https://github.com/Day8/re-frame-trace) with the intention to make it more easily usable. The first things we focused on were improving the filter functionality, making resizing of the window smooth and easy and adding some adjustments to the user interface. Another important addition was saving user data like filters to localstorage to ensure persistence. The traces that are being shown in the panel are still a lot and can feel a bit overwhelming. Our focus for the next weeks is to reduce the amount of traces to the ones that are important. For the end of the summer, we want compare the tracing tool before and after we started working on it.
```
Comment (v-thakkar) [inline blog/_posts/2017-09-04-clojurians-update.md]: Change 'we want compare' -> 'we want **to** compare'

File: blog/_posts/2017-09-04-clojurians-update.md [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+
+## Community <3
+
+In the third week of our project, we went to **EuroClojure**, which took place in Berlin this year. 
+
+[at EuroClojure](/img/blog/2016/euroclojure.png)
+*photo taken by Chris Sun* 
+
+It was a great opportunity to connect to people from around the globe some of which are using re-frame in their everyday work. Being at conferences always leaves me very inspired and full of ideas for what to look into next. We also got some feedback and ideas for how re-frame’s developer experience could be improved. In general, people were really excited about our project and some of them met us for lunch in the following weeks to talk about the project in more depth. These meetings were super valuable for us, as we were able to focus on very specific parts of development with re-frame. It's so wonderful to be surrounded by the community in person after chatting over slack and github and twitter!
```
Comment (v-thakkar) [inline blog/_posts/2017-09-04-clojurians-update.md]: Change 'always leaves **me**' -> 'always leaves **us**'

File: blog/_posts/2017-09-04-clojurians-update.md [hunk 4]
```diff
@@ -0,0 +1,64 @@
+---
+title: Clojurian's Update - Working on re-frame's Tracing Tool
```
Comment (v-thakkar) [inline blog/_posts/2017-09-04-clojurians-update.md:2]: Hi, thanks for sending a PR for your 2nd blog post. Check out the comments below. :)

File: blog/_posts/2017-09-04-clojurians-update.md [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+
+## Learning and Growing
+
+Through a glorious combination of working on re-frame's tracing tool, reading other peoples' open source projects, hanging out with our wonderful mentors and coaches, and tinkering with our own side projects, we gathered experience in programming with ClojureScript and re-frame and learned about langugage specific idioms. What I find most valuable though is the great advice we got from our coaches and our mentor. They all have given us so much input for how to become good programmers, not only technically, but especially in terms of finding a good and efficient workflow, addressing problems on a conceptual level and the importance of clear and constructive communication. 
+
+[coaching with Matt!](/img/blog/2016/coaching.jpg)
+*photo taken by Chris Sun*
```
Comment (v-thakkar) [inline blog/_posts/2017-09-04-clojurians-update.md]: You would need to use image credit class for the images. You can check out the other posts for an example. Can you please change this for all the images?

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2017-09-04-clojurians-update.md (hunk 4 above)
Comment (saskali) [reply blog/_posts/2017-09-04-clojurians-update.md:2]: thanks for reviewing, I'll make the adjustments in a bit :)

File: blog/_posts/2017-09-04-clojurians-update.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+
+## Learning and Growing
+
+Through a glorious combination of working on re-frame's tracing tool, reading other peoples' open source projects, hanging out with our wonderful mentors and coaches, and tinkering with our own side projects, we gathered experience in programming with ClojureScript and re-frame and learned about langugage specific idioms. What we find most valuable though is the great advice we got from our coaches and our mentor. They all have given us so much input for how to become good programmers, not only technically, but especially in terms of finding a good and efficient workflow, addressing problems on a conceptual level and the importance of clear and constructive communication. 
+
+![image](/img/blog/2016/coaching.jpg)
```
Comment (v-thakkar) [inline blog/_posts/2017-09-04-clojurians-update.md]: Sorry, I forgot to mention in the previous review that image locations are wrong as well. All your images are in [/img/blog/2017/] folder. Can you please fix these links as well?

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2017-09-04-clojurians-update.md (hunk 6 above)
Comment (saskali) [reply blog/_posts/2017-09-04-clojurians-update.md]: oh true, I changed them to the correct location!

File: blog/_posts/2017-09-04-clojurians-update.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+
+## Building and Collaborating
+
+We worked on many features for the [tracing tool](https://github.com/Day8/re-frame-trace) with the intention to make it more easily usable. The first things we focused on were improving the filter functionality, making resizing of the window smooth and easy and adding some adjustments to the user interface. Another important addition was saving user data like filters to localstorage to ensure persistence. The traces that are being shown in the panel are still a lot and can feel a bit overwhelming. Our focus for the next weeks is to reduce the amount of traces to the ones that are important. For the end of the summer, we want to compare the tracing tool before and after we started working on it.  
+
+## Community <3
+
+In the third week of our project, we went to **EuroClojure**, which took place in Berlin this year. 
+
+![image](/img/blog/2017/euroclojure.png)
```
Comment (v-thakkar) [inline blog/_posts/2017-09-04-clojurians-update.md:35]: Thanks for the quick fixes but there is one more issue. Two of these image files are of .jpeg formats and not .png. I thought you'll catch it but no issues. You can fix them in the next commit. :)

**Review by v-thakkar — CHANGES_REQUESTED** [review]

File: blog/_posts/2017-09-04-clojurians-update.md (hunk 7 above)
Comment (saskali) [reply blog/_posts/2017-09-04-clojurians-update.md:35]: thank you, I pushed the changes!

**Review by v-thakkar — APPROVED** [review]

**Pull request merged**